packages/custom-decorators/test/is-enum-decorator.test.ts

```diff
@@ -0,0 +1,79 @@
+import { IsEnum } from '../src/is-enum.decorator';
+import { Validator, ValidatorOptions } from 'class-validator';
+
+function validateValues(
+  object: { testProperty: any },
+  values: any[],
+  validatorOptions?: ValidatorOptions,
+): Promise<any> {
+  const validator = new Validator();
+  const promises = values.map((value) => {
+    object.testProperty = value;
```

Comment:
there is just one instance of object but you set and test many async validation parallelly on the same property, I'm not sure it works correctly as expected